packages/google_atlas/lib/src/utils/lat_lng_bounds_utils.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:atlas/atlas.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart' as GoogleMaps;
+
+class LatLngBoundsUtils {
+  static GoogleMaps.LatLngBounds mapMarkersToLatLngBounds(Set<Marker> markers) {
+    double maxNorth = -90.0;
```

Comment:
Should we extract all these values to private constants for clarification? (there's plenty of these in this class). Do you think it would read better, or it is not worth it?

Reply:
Also, can we add docs to these methods? It is not very clear what they are doing...